components/card/package.json

```
@@ -24,17 +24,17 @@
     "@spectrum-css/icon": ">=3",
     "@spectrum-css/quickaction": ">=3",
     "@spectrum-css/typography": ">=4 <=5",
-    "@spectrum-css/vars": ">=9"
+    "@spectrum-css/tokens": ">=8 <=9"
```

```suggestion
    "@spectrum-css/tokens": ">=8 <=10"
```

I believe the token package is up to 10 now.